docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -1279,7 +1279,10 @@ The following settings are available:
 -   `size`: The maximum cache size in bytes. 0 means the query cache is disabled. Default value: `1073741824` (1 GiB).
 -   `max_entries`: The maximum number of `SELECT` query results stored in the cache. Default value: `1024`.
 -   `max_entry_size`: The maximum size in bytes `SELECT` query results may have to be saved in the cache. Default value: `1048576` (1 MiB).
--   `max_entry_records`: The maximum number of records `SELECT` query results may have to be saved in the cache. Default value: `30000000` (30 mil).
+-   `max_entry_rows`: The maximum number of rows `SELECT` query results may have to be saved in the cache. Default value: `30000000` (30 mil).
+-   `compress_entries`: If cache entries are compressed. Reduces the cache memory consumption but reduces cache insert/read performance. Default value: `true`.
```

Comment:
Another idea is to have the compressed flag near each entry, and then the compression can be controlled by a query-level setting on the side of the query writing to the cache.

Reply:
Sounds good, can do.